specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/analyzeconversations.json

```diff
@@ -290,43 +316,72 @@
           "type": "string",
           "description": "The conversation utterance given by the caller."
         },
+        "detectedLanguage": {
+          "type": "string",
+          "description": "The system detected language for the query."
+        },
         "prediction": {
           "description": "The prediction result of a conversation project.",
-          "$ref": "#/definitions/AnalyzePrediction"
+          "$ref": "#/definitions/BasePrediction"
         }
       }
     },
-    "AnalyzePrediction": {
+    "BasePrediction": {
+      "description": "This is the base class of prediction",
       "required": [
-        "topIntent",
-        "intents"
+        "projectType"
       ],
-      "description": "Represents the prediction section in the response body.",
-      "type": "object",
+      "discriminator": "projectType",
       "properties": {
-        "topIntent": {
-          "description": "The name of the top scoring intent.",
-          "type": "string"
+        "projectType": {
+          "type": "string",
+          "description": "The type of the project: either conversation or workflow. Use 'conversation' For LUIS deepstack projects, and workflow for Arbitration projects.",
```

Comment:
Given the acceptable values are below, I wouldn't mention them in the description as well. Redundant, and more to keep in sync. I would instead move the descriptions of each under x-ms-enum, which you can add descriptions. I don't have an example off hand, but you can find some, IIRC, for questionanswering.